patches/content-renderer-render_frame_impl.cc.patch

```diff
@@ -1,8 +1,61 @@
 diff --git a/content/renderer/render_frame_impl.cc b/content/renderer/render_frame_impl.cc
-index d7f19a4efc3540d8bac48d00c37436362043f074..5b1e662bf87994848264716c939478ae3b98d49d 100644
+index d7f19a4efc3540d8bac48d00c37436362043f074..832feda8445ddba637f6e65a15b170669586db83 100644
 --- a/content/renderer/render_frame_impl.cc
 +++ b/content/renderer/render_frame_impl.cc
-@@ -4422,6 +4422,7 @@ bool RenderFrameImpl::ShouldUseUserAgentOverride() const {
+@@ -581,8 +581,51 @@ blink::mojom::CommonNavigationParamsPtr MakeCommonNavigationParams(
+       info->url_request.RequestorOrigin().CanAccess(current_origin),
+       has_download_sandbox_flag, from_ad);
+ 
++  GURL url = info->url_request.Url();
```

Comment:
This is another way to get the renderer-initiated navigations to work (currently commented-out).